gax-grpc/src/test/java/com/google/api/gax/grpc/GaxGrpcPropertiesTest.java

```diff
@@ -40,8 +41,14 @@
 public class GaxGrpcPropertiesTest {
 
   @Test
-  public void testGrpcVersion() throws Exception {
+  public void testGrpcVersion() {
     String grpcVersion = GaxGrpcProperties.getGrpcVersion();
     assertTrue(Pattern.compile("^\\d+\\.\\d+\\.\\d+").matcher(grpcVersion).find());
   }
+
+  @Test
+  public void testGaxGrpcVersion() {
+    String gaxGrpcVersion = GaxGrpcProperties.getGaxGrpcVersion();
+    assertNotNull(gaxGrpcVersion);
```

Comment:
Why no pattern matching on the grpc version?

Reply:
Because it will be empty in case if run from sources. This is by design and I believe is absolutely ok (please check other comment about `PropertiesProvider` for more details).